image/image.go

```diff
@@ -380,6 +409,15 @@ func bootstrapToRootDir(tsto *ToolingStore, model *asserts.Model, opts *Options,
 			fmt.Fprintf(Stdout, "Fetching %s\n", snapName)
 		}
 
+		dlOpts.Channel = opts.Channel
+		if snapName == model.Kernel() && model.KernelTrack() != "" {
```

Comment:
needing this in two places seems a bit strange now.  Afair info.Channel is only used below to set Channel in the seed. So we can remove the code in localSnaps, and have only this one and set Channel below form a local in loop snapChannel var. Also given this code we probably should make dlOpts on the fly at each loop, seems cleaner.

Reply:
to clear, I'm saying that in retrospect we don't need  any info.Channel = at all in localSnaps